### ISSUE TITLE
Dropping Windows, at least temporarily?

### ISSUE THREAD

**Issue by homonoidian** [novika-lang/novika#44]
0.0.4 *could* be the last version with Windows support.

I'm only learning advanced UI for a world-like environment, so I guess I won't make one tomorrow. I do believe I'll have to write it in Crystal, because Novika is just too slow for UI unless we have a super-advanced e.g. screen damage engine (for one of the most intensive parts of any graphical application is drawing frames; I'm talking about doing that with the CPU here of course).

Making tiny console tools using a console framework written in Novika sounds better, but that unfortunately would mean dropping Windows at least temporarily, as I haven't found any way to e.g. get terminal size on both Windows and Linux due to some weird linking issue (maybe it's my fault though). The thing is, I don't want to depend on termbox2 because it's acting weird on Unicode, but do terminals support at least partial Unicode at all? Novika assumes screen support for at least *a lot* of Unicode characters. Displaying them with a UI rather than a TUI should be easier, although on the other hand, aterminal app may help by providing a nice font stack with a lot of characters, and with no effort for us.

TL;DR: I'll really have to consider this, unless it's nuclear apocalypse or I finally write something that could function as a browser of sorts, good enough for general Novika usage.

**Comment by homonoidian** [comment]
With my promising signal (think of SolidJS)/declarative DSL/SDL experiments, I don't think dropping Windows is something we should worry about now. Soon (like months soon) there will be something GUIsh, and this something would then be able to serve as the basis of Novika block browser.
